src/fixate/drivers/dmm/helper.py

```diff
@@ -60,10 +61,14 @@ def voltage_dc(self, _range=None, auto_impedance=False):
         """
         raise NotImplementedError
 
-    def current_ac(self, _range):
+    def current_ac(self, _range, port: Literal["HIGH", "LOW"]):
```

Comment:
If we're going to introduce braking changes, maybe now is a good opportunity to make the not so private private `_range` not private anymore?

Reply:
I did have this thought, then it annoyed me that only one function in the whole API uses the correct naming convention. I know it's wrong, but I almost prefer to have everything consistent? Let me know if you really want me to change it.